src/EnergyPlus/Furnaces.cc

```diff
@@ -7587,107 +7587,99 @@ namespace Furnaces {
-                            //         OnOffAirFlowRatio is updated during the above iteration. Reset to correct value based on PLR.
-                            OnOffAirFlowRatio = state.dataFurnaces->OnOffAirFlowRatioSave;
-                            if (SolFlag < 0) {
-                                if (SolFlag == -1) {
-                                    CalcFurnaceOutput(state,
-                                                      FurnaceNum,
-                                                      FirstHVACIteration,
-                                                      OpMode,
-                                                      CompressorOp,
-                                                      0.0,
-                                                      PartLoadRatio,
-                                                      0.0,
-                                                      0.0,
-                                                      TempHeatOutput,
-                                                      TempLatentOutput,
-                                                      OnOffAirFlowRatio,
-                                                      false);
-                                    if (std::abs(SystemSensibleLoad - TempHeatOutput) > SmallLoad) {
-                                        if (state.dataFurnaces->Furnace(FurnaceNum).DXHeatingMaxIterIndex == 0) {
-                                            ShowWarningMessage(state,
-                                                               format("Heating coil control failed to converge for {}:{}",
-                                                                      cFurnaceTypes(state.dataFurnaces->Furnace(FurnaceNum).FurnaceType_Num),
-                                                                      state.dataFurnaces->Furnace(FurnaceNum).Name));
-                                            ShowContinueError(state,
-                                                              "  Iteration limit exceeded in calculating DX heating coil sensible part-load ratio.");
-                                            ShowContinueErrorTimeStamp(
-                                                state,
-                                                format("Sensible load to be met by DX heating coil = {:.2T} (watts), sensible output of DX heating "
-                                                       "coil = {:.2T} (watts), and the simulation continues.",
+                        // Calculate the part load ratio through iteration
+                        HeatErrorToler =
+                            state.dataFurnaces->Furnace(FurnaceNum).HeatingConvergenceTolerance; // Error tolerance for convergence from input deck
+
+                        SolFlag = 0; // # of iterations if positive, -1 means failed to converge, -2 means bounds are incorrect
+                        Par[0] = double(FurnaceNum);
```

Comment:
Why is this `Par` stuff still here when we are using a lambda below?

Reply:
What caught my eye was the comments in the lambda (par6, 7, 9, 10) versus the comments from the previous Par variables (Par 5, 6, 8, 9). That's probably a cut-n-paste thing so these can be safely deleted.

     0.0,  // par6_loadFlag,
     1.0,  // par7_sensLatentFlag,
     0.0,  // par9_HXOnFlag,
     0.0); // par10_HeatingCoilPLR);

     Par[5] = 0.0;               // FLAG, 0.0 if heating load, 1.0 if cooling or moisture load
     Par[6] = 1.0;               // FLAG, 0.0 if latent load, 1.0 if sensible load to be met
     Par[8] = 0.0;               // HXUnitOn is always false for HX
     Par[9] = 0.0;